EventFilter/L1TRawToDigi/plugins/implementations_stage2/GTSetup.cc

```diff
@@ -116,6 +122,7 @@ namespace l1t {
         res[16] = tau_unp;
         res[18] = tau_unp;
         res[20] = etsum_unp;
+        res[22] = zdc_unp;
```

Comment:
I suppose it's confirmed that ZDC inputs come on link 11 of the uGT?

Reply:
I understood it was link 71 (0x8E)

Reply:
I was not sure of the link. If the link is 71, then it should be res[142]?

Reply:
If the logical link ID is 71, then yes I think res[142] should be correct. 
Cross referencing the GT input data format [1] with the unpacker [2] I think supports this. 

[1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/GlobalTriggerInputDataFormat
[2] https://github.com/cms-sw/cmssw/blob/440318ca7e69a9a79fcd262a9deafb54bb933916/EventFilter/L1TRawToDigi/plugins/implementations_stage2/GTSetup.cc#L114

Reply:
(I think even links is Rx and odd links is Tx)

Reply:
Yeah, my understanding is also that input ports get even numbers, so it would be 142 if it's link 71.